pinot-core/src/main/java/org/apache/pinot/core/startree/StarTreeUtils.java

```diff
@@ -258,7 +258,7 @@ private static boolean extractOrClausePredicates(FilterContext filter, List<Pred
    */
   @Nullable
   private static PredicateEvaluator getPredicateEvaluator(IndexSegment indexSegment, Predicate predicate,
-      Map<Predicate, PredicateEvaluator> predicateEvaluatorMap) {
+      List<Pair<Predicate, PredicateEvaluator>> predicatesAndEvaluators) {
```

Comment:
(nit) for consistency
```suggestion
      List<Pair<Predicate, PredicateEvaluator>> predicateEvaluators) {
```